client/src/node_api/participation/types/participation.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{convert::TryInto, io::Read};
+
+use serde::{Deserialize, Serialize};
+
+/// Participation information.
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+pub struct Participation {
+    /// A staking or voting event id, hex encoded [u8; 32]
+    #[serde(rename = "eventId")]
+    pub event_id: String,
+    /// Answers for a voting event, can be empty
+    pub answers: Vec<u8>,
+}
+
+/// Participation information.
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+pub struct Participations {
+    /// Multiple participations that happen at the same time
+    pub participations: Vec<Participation>,
+}
+
+impl Participations {
+    // https://github.com/alexsporn/treasury/blob/main/specifications/chrysalis-referendum-rfc.md#structure-of-the-participation
+    /// Serialize to bytes
+    pub fn to_bytes(&self) -> crate::Result<Vec<u8>> {
```

Comment:
Can this not be Packable?

Reply:
I'll create an issue to think about it later, but it would be nice to be consistent and have Packable here.

Reply:
https://github.com/iotaledger/iota.rs/issues/1363